app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -76,12 +78,19 @@ def destroy
 
   private
 
+  def rename_config_template
+    if params[taxonomy_single] && params[taxonomy_single][:config_template_ids].present?
+      params[taxonomy_single][:provisioning_template_ids] = params[taxonomy_single].delete(:config_template_ids)
+      ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
+    end
+  end
+
   def params_match_database
     # change params[:select_all_types] to params[:ignore_types] to match database
-    if params[taxonomy_single.to_sym].try(:[], :select_all_types)
-      params[taxonomy_single.to_sym][:ignore_types] = params[taxonomy_single.to_sym][:select_all_types]
-      params[taxonomy_single.to_sym]                = params[taxonomy_single.to_sym].reject { |k, v| k == "select_all_types" }
-      return params[taxonomy_single.to_sym]
+    if params[taxonomy_single].try(:[], :select_all_types)
+      params[taxonomy_single][:ignore_types] = params[taxonomy_single][:select_all_types]
+      params[taxonomy_single]                = params[taxonomy_single].reject { |k, v| k == "select_all_types" }
+      return params[taxonomy_single]
```

Comment:
@eLobato as @ares suggests I'd prefer to touch the unrelated stuff as little as possible.